docs/recommendations.md

```diff
@@ -2,6 +2,8 @@
 
 ### _Early draft for discussion_
 
+## Description of the experiment and crystal
+
 Most of the definitions in the core CIF dictionary can be applied to a description of a 3D ED structure determination experiment. An example of good practice in describing the relevant experimental metadata follows, with some guidance for individual data items. (The current example is adapted from the published structure of Andrusenko, I., Potticary, J., Hall, S. R. & Gemmi, M. (2020). _Acta Cryst._ B**76**, 1036-1044, and contains some deviations from the later recommendations. An ideal example will be substituted at a later date.)
 
 ```
```

Comment:
I example below is slightly outdated. I cannot mark the specific lines so a bullet list will have to do:
- `_diffrn_precession.angle` should be changed to `_diffrn.precession_semi_angle`.
- `_diffrn_measurement.integration` should be changed to `_diffrn_measurement.rotation_mode`.
- `_diffrn_radiation.wavelength` should be changed to `_diffrn_radiation_wavelength.value` (`_diffrn_radiation.wavelength` alias is not currently defined in the CIF_CORE dictionary).

Also, section "`_diffrn_source.voltage` and `_diffrn_radiation.wavelength`" of this document claim that the `_diffrn_radiation.wavelength` is expected to include SU so it would be nice to also do that in the given example if possible.